microsoft/PowerToys
# Localize PowerRename

## Summary of the Pull Request
This pull request is to localize the PowerRename Project. All the hardcoded strings from the PowerRename projects are moved to resource script files. 

 
## References


## PR Checklist
* [x] Applies to #199 
* [x] PowerRename Tests passed


## Detailed Description of the Pull Request / Additional comments
All hardcoded strings have been moved out to the .rc (resource script files) of each project. Wrappers have been implemented to extract these resources from the string tables of the resource script files.


## Validation Steps Performed
* Build and install all configurations of PowerToys (Debug/Release x64)
* All Unit tests pass
* Installed PowerToys run as expected.
